packages/cerebral-forms/package.json

```diff
@@ -26,7 +26,9 @@
     "cerebral": "^2.0.0-0"
   },
   "devDependencies": {
-    "cerebral": "^2.0.0-0"
+    "cerebral": "^2.0.0-0",
+    "babel-preset-es2015": "^6.16.0",
```

Comment:
aaah i see now your comment. i need to put it in scripts :)
thanks @Guria, i'm start to slowly understand this npm ecosystem :)